docs/io/ascii/read.rst

```diff
@@ -23,6 +23,7 @@ format, for example::
    >>> data = astropy.io.ascii.read('t/nls1_stackinfo.dbout', data_start=2, delimiter='|')  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple.txt', quotechar="'")  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple4.txt', format='no_header', delimiter='|')  # doctest: +SKIP
+   >>> data = astropy.io.ascii.read('t/tab_and_space.txt', delimiter='\s')  # doctest: +SKIP
```

Comment:
I had to look up what `\s` means. All the delimiters in the other examples are obvious. I'm sure the python experts know, but I've been using python for 7+ years now and I did not...
Thus, I suggest a small note like "(delimiters can be ascii characters or special thingies[link to the list of special thingies in the python docs] )".